velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -485,6 +485,119 @@ TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
           "year(c0)", std::nullopt, std::nullopt));
 }
 
+TEST_F(DateTimeFunctionsTest, weekDate) {
+  const auto week = [&](std::optional<Date> date) {
+    return evaluateOnce<int64_t>("week(c0)", date);
+  };
+  auto parseDate = [&](const StringView& dateStr) {
```

Comment:
This lambda repeats multiple time. Consider promoting it to a method of DateTimeFunctionsTest to avoid code duplication.

Reply:
Done, thanks!